CHANGELOG.md

```diff
@@ -4,6 +4,32 @@
 
 ## [Unreleased]
 
+## [0.1.0]
+
+_2023-04-18_
+
+### What's new since 0.1.0
```

Comment:
Is this auto-generated? Because `since 0.1.0` does not make sense since it is the newly released version